### DISCUSSION TITLE
That source is fallible

### DISCUSSION THREAD

**Discussion by imehesz** [savonet/liquidsoap#2395] [User Support]
hello,

I'm new to Liquidsoap, but I cannot get even the most simplistic example to work. I have an Ubuntu (20.04) box and I'm trying to make this a dedicated radio station. I have the following liq file:

![image](https://user-images.githubusercontent.com/13553/167298955-51e704f7-c197-4686-9a11-69881c4045b3.png)

and I'm getting this error (constantly)

![image](https://user-images.githubusercontent.com/13553/167299021-72310667-1e29-4166-85ae-2cc09ee25ef5.png)

This should work right? (There is an icecast2 with a mount point setup already)

**Comment by cdgraff** [comment]
Hi @imehesz, you need to read the explanation as FALLIBLE source, that is one of the core concepts using Liquidsoap.

I recommend read here: https://www.liquidsoap.info/doc-2.0.4/quick_start.html

And please don't share images, the recommended way is share text, with full log and full code, you can paste here, or using gist. 

Your code is not complete and for that reason is hard to share some snipped of code to fix your error. But basically you are missing the FALLIBLE true/false in your input, as I mention that is a core concept explained into the link shared above. regards.
